peps/pep-0774.rst

```diff
@@ -220,9 +221,7 @@ process. Additional tooling would be required to fetch the stencils and
 potentially create additional and unnecessary failure points in the workflow.
 This separation also makes it harder to ensure consistency between the stencils
 and the CPython source tree, as updates must be coordinated across the
-repositories. Finally, this approach introduces an attack vector, as external
```

Comment:
I missed removing this section before publish. It's stale and actually a non-concern in this case.